assets/gdrive/manifest.yml

```diff
@@ -1,49 +1,56 @@
+# yaml-language-server: $schema=https://www.topaz.sh/schema/manifest.json
+---
+
+### filename: manifest.yaml ###
+### datetime: 2023-12-19T00:00:00-00:00 ###
+### description: gdrive manifest ###
+#
 # model
 model:
   version: 3
 
 # object type definitions
 types:
-  # user represents any rebel that has access to the system
-  user: {}
-
-  # group represents a collection of users and/or (nested) groups
-  group:
+  ### display_name: User ###
+  user:
     relations:
-      member: user | group#member
+      ### display_name: user#manager ###
+      manager: user
 
-  # author represents a user or member of a group that can create a document
-  author:
+  ### display_name: Identity ###
+  identity:
     relations:
-      member: user | group#member
+      ### display_name: identity#identifier ###
+      identifier: user
 
-    permissions:
-      can_create: member
+  ### display_name: Group ###
+  group:
+    relations:
+      ### display_name: group#member ###
+      member: user
 
   # folder represents a collection of documents and/or other folders
   folder:
     relations:
       owner: user
       editor: user | group#member
       viewer: user | group#member
-      author: user | group#member
       parent: folder
     permissions:
       can_view: viewer | editor | owner | parent->can_view
       can_write: editor | owner | parent->can_write
       can_share: owner | parent->can_share
-      can_create: parent->can_create
 
   # doc represents a document within a folder
   doc:
     relations:
-      owner: user 
+      owner: user
       editor: user | group#member
       viewer: user | group#member
       parent: folder
 
     permissions:
-      can_view: viewer | editor | owner | parent->can_view 
+      can_view: viewer | editor | owner | parent->can_view
```

Comment:
I think we should use `can_read` for permission names and `viewer` for relation names. This reduces the risk of confusing the relation names and permission names, and also is more consistent with `can_write` and `editor`.

Reply:
Renamed to can_read.